docs/getting-started/rulesets.md

```diff
@@ -231,6 +231,15 @@ The example above will run the recommended rules from the `spectral:oas2` rulese
 
 Available severity levels are `error`, `warn`, `info`, `hint`, and `off`.
 
+## Enriching rule messages
+
+To help you create meaningful error messages, Spectral comes with a couple of placeholders that are evaluated at runtime.
+
+- {{error}} - the error returned by function
```

Comment:
Can we add an example of a rule which has an awesome message using some of these variables to tie it all together.

Reply:
Sure, sounds like a good idea. Will do it in a bit.